src/main/java/com/treasuredata/client/TDClientApi.java

```diff
@@ -176,6 +176,14 @@
      */
     void createTable(String databaseName, String tableName);
 
+    /**
+     * EXPERIMENTAL: Create a new table with idempotent retry
+     * @param databaseName
+     * @param tableName
+     * @param idempotentKey
+     */
+    void createTable(String databaseName, String tableName, String idempotentKey);
+
     void createTableIfNotExists(String databaseName, String tableName);
```

Comment:
don't we need `createTableIfNotExists(String databaseName, String tableName, String idempotentKey)` ?

Reply:
Since `createTableIfNotExists` ignores the `TDClientHttpConflictException`, it does not throw error even the table already exists. I think we do not need to use idempotent key here.

Reply:
ok good